pkgs/development/lisp-modules/lisp-packages.nix

```diff
@@ -194,7 +194,7 @@ let lispPackages = rec {
       owner = "atlas-engineer";
       repo = "nyxt";
       rev = "${version}";
-      sha256 = "sha256-eSRNfzkAzGTorLjdHo1LQEKLx4ASdv3RGXIFZ5WFIXk=";
+      sha256 = "sha256:0kzm05swhyb197cjfd3iglf60b997sx7v95yxzyq483jxqbcxm0r";
```

Comment:
```suggestion
      sha256 = "0kzm05swhyb197cjfd3iglf60b997sx7v95yxzyq483jxqbcxm0r";
```

Reply:
???

Reply:
I think the long-term plan is SRI hashes everywhere, so moving away from SRI seems not necessary?.

Reply:
I simply pasted what Nix reported to me on error. If I should not have "sha256:" in the string, that is fine with me. I don't see the difference really.

Reply:
None sri hashes should not have the prefix. SRI hashes are allowed to have them.

Reply:
Wait, what is «allowed» here? They won't work without it if you mean `hash = `, and they won't be in SRI format without prefix in `sha256 =` case